rmc/utils/mpc.py

```diff
@@ -605,22 +621,10 @@ def calculate_fitted_scores(
     ]
     interaction_annot_expr = []
     for variable in interactions_dict.keys():
```

Comment:
I think renaming `variable` to `interaction` would be more clear here — and then rename `var_split` to `variable` on 625/626

Reply:
Done